### PR TITLE
Remove gcc specific files.

### PR DESCRIPTION
**Pull request by honekun**
Hi Adam, Nelson,

I have copied the GCC version of the lean theme into the repo and removed the gcc specific files I could find.
Could you help me review the updates and let me know if more files can be removed.

Thanks!

### CODE REVIEW

File: backend/WP/Images.php [hunk 1]
```diff
@@ -18,24 +18,42 @@ public static function init() {
...
+			'desktop@2x' => 2400,
+		];
+
 		$ratios = [
 			'portrait'       => [
-				'ratio'  => 3 / 4,
+				'ratio'  => 3 / 4, // 0.75
+				'widths' => $default_widths,
+			],
+			'portrait-hero'  => [
```
Comment (adamf321) [inline backend/WP/Images.php]: Can remove this one

**Comment by adamf321** [comment]
@honekun I've commented on a few things. Overall I think it's best to include most of this code and then strip it out in the NYL theme for now.
Also you have a TravisCI error.

File: ThemeSetup.php [hunk 2]
```diff
@@ -171,9 +152,6 @@ public static function init_gutenberg_config() {
 
 		// Wide/Fullwidth images.
 		add_theme_support( 'align-wide' );
-
-		// Responsive embeds.
-		add_theme_support( 'responsive-embeds' );
```
Comment (nelsonamaya82) [inline ThemeSetup.php]: @honekun why did you remove responsive-embeds support? I think we should preserve it.

File: backend/WP/Gutenberg/AbstractBlock.php [hunk 3]
```diff
@@ -101,7 +101,7 @@ public function render( array $block, string $content, bool $is_preview, int $po
 				[
 					'block',
 					'block-' . $this->get_name(),
-					'has-text-align-' . $block['align'] ?? 'left',
+					'has-text-align-' . ( '' === $block['align'] ? 'left' : $block['align'] ),
```
Comment (nelsonamaya82) [inline backend/WP/Gutenberg/AbstractBlock.php]: I think this does exactly the same as the removed line, but longer. We could keep the previous version of this (unless you think it doesn't work).

File: frontend/components/atoms/animations/scroll.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/* purgecss start ignore */
```
Comment (nelsonamaya82) [inline frontend/components/atoms/animations/scroll.css]: I'm not sure if we should keep scrolling animations by default in the Lean Theme base, what do you think @adamf321 , @Fcog ?

File: frontend/components/atoms/images/main-logo.php [hunk 5]
```diff
@@ -18,7 +18,7 @@
 		);
 		?>
 	<?php else : ?>
-		<div class="inline font-bold uppercase text-small">
+		<div class="inline font-bold uppercase text-1.4">
```
Comment (nelsonamaya82) [inline frontend/components/atoms/images/main-logo.php]: I think this 1.4 is too specific, maybe we should keep it as text-small and then modify it based on each project requirements.

File: frontend/components/organisms/heroes/media-small.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+
+{
+   "variants": [
+      {
+        "images": [ "${image-id}" ],
+        "title": "Cement Products Breadcrumbs",
```
Comment (nelsonamaya82) [inline frontend/components/organisms/heroes/media-small.json]: I'd suggest changing this dummy text to something more generic like Lorem Ipsum text.

File: frontend/tailwind.config.js [hunk 7]
```diff
@@ -138,19 +175,41 @@ module.exports = {
 		},
 		fontFamily: {
 			'primary': [
-				'sans-serif',
+				'neue-haas-unica',
+        '-apple-system',
+        'BlinkMacSystemFont',
```
Comment (nelsonamaya82) [inline frontend/tailwind.config.js]: I think we should cleanup this file by removing some project-specific styles like font families, just keeping some standard fonts like Arial or something like that.

File: frontend/tailwind.config.js [hunk 8]
```diff
@@ -196,16 +277,20 @@ module.exports = {
...
 			auto: 'auto',
+			'screen': 'calc(50% - 50vw)',
 			...theme('spacing'),
 			...negative(theme('spacing'))
 		}),
 		maxHeight: {
 			full: '100%',
-			screen: '100vh'
+			screen: '100vh',
+			'37': '37rem',
```
Comment (nelsonamaya82) [inline frontend/tailwind.config.js]: Also we should remove this project-specific styles.

File: frontend/tailwind.config.js [hunk 9]
```diff
@@ -214,7 +299,11 @@ module.exports = {
 			'0': '0',
 			full: '100%',
 			screen: '100vh',
-			auto: 'auto'
+			auto: 'auto',
+			'24.38': '24.375rem',
```
Comment (nelsonamaya82) [inline frontend/tailwind.config.js]: Same here.

File: backend/WP/Gutenberg/AbstractBlock.php (hunk 3 above)
Comment (honekun) [reply backend/WP/Gutenberg/AbstractBlock.php]: I think this was added for cases when $block['align'] exists but is an empty string, since the ?? operator only checks for NULL

File: ThemeSetup.php (hunk 2 above)
Comment (honekun) [reply ThemeSetup.php]: Agree, I'll add it back

File: frontend/components/organisms/heroes/media-small.json (hunk 6 above)
Comment (honekun) [reply frontend/components/organisms/heroes/media-small.json]: Yes, I'll update this

File: frontend/components/atoms/images/main-logo.php (hunk 5 above)
Comment (honekun) [reply frontend/components/atoms/images/main-logo.php]: I'll update

File: frontend/tailwind.config.js (hunk 7 above)
Comment (honekun) [reply frontend/tailwind.config.js]: I'll remove the project-specific fonts

File: frontend/tailwind.config.js (hunk 9 above)
Comment (honekun) [reply frontend/tailwind.config.js]: I'll remove them

File: frontend/tailwind.config.js (hunk 8 above)
Comment (honekun) [reply frontend/tailwind.config.js]: I'll remove them

File: backend/WP/Images.php (hunk 1 above)
Comment (honekun) [reply backend/WP/Images.php]: I'll remove them

File: frontend/components/atoms/animations/scroll.css (hunk 4 above)
Comment (honekun) [reply frontend/components/atoms/animations/scroll.css]: I'll keep this for the moment
